### PR TITLE
Install instructions for Windows - update

### PR DESCRIPTION
**Pull request by flor14**
Changes made to **install instructions for Windows**:
* Updated software versions
* LaTeX installation for Jupyter was changed. I was unable to install it based on the pre-existing instructions due to a broken link.
* Instructions to use the new native pipe operator  `|>` as a shortcut were included/updated in RStudio and Jupyter Lab.
* "PostgreSQL should not be installed in version 13" was removed.
* Minor changes in format.

@joelostblom

### CODE REVIEW

File: resources_pages/install_ds_stack_windows.md [hunk 1]
```diff
@@ -538,39 +547,34 @@ named COPYING and the pdfTeX source.
...
-To test that your latex installation is working with jupyter notebooks,
-launch `jupyter lab` from a terminal and open either a new notebook
-or the same one you used to test IRkernel above.
+Compiled with xpdf version 4.03
+```
+
+The above is all we need to have LaTeX work with R Markdown documents. However, for Jupyter we need to add several more packages. To complete the installation open an R console in JupyterLab, type the commands listed below and press `Ctrl` + `enter`:
+
+```
+tinytex::tlmgr_install(c("eurosym",
```
Comment (joelostblom) [inline resources_pages/install_ds_stack_windows.md]: Why did you change this to run in the R console instead of a terminal? Does it have any advantage? I vaguely remember issues with not seeing the path correctly unless this was run from terminal.

File: resources_pages/install_ds_stack_windows.md [hunk 2]
```diff
@@ -491,7 +498,7 @@ To improve the experience of using R in JupyterLab,
 we will add an extension that allows us to setup keyboard shortcuts for inserting text
 (thanks to former MDS student Ryan Homer for developing this extension!).
 By default,
-it creates shortcuts for inserting two of the most common R operators: `<-` and `%>%`.
+it creates shortcuts for inserting two of the most common R operators: `<-` and `|>`.
```
Comment (joelostblom) [inline resources_pages/install_ds_stack_windows.md:499]: Could you make changes like this in all three instructions (mac, win, linux?), so that they don't go out of sync with eachother and we forget one of them.

File: resources_pages/install_ds_stack_windows.md [hunk 3]
```diff
@@ -426,14 +426,21 @@ Click "OK" on all of the three windows we opened above and you're done! If you o
...
-"C:/Users/joelo/R/win-library/4.0"   "C:/Program Files/R/R-4.0.2/library"
+"C:/Users/florencia/R/win-library/4.1" "C:/Program Files/R/R-4.1.0/library"
 ```
 
 If they don't return the same paths, please try to setting up your environmental variable again
 and making sure that it is pointing to the correct folder.
 
 **Do not continue unless both R from terminal and R from RStudio return the same paths here or later parts of the installation will fail.**
 
+Now we are going to change RStudio’s *Insert Pipe* shortcut so that it inserts the [new native pipe operator `|>`](https://blog.rstudio.com/2021/06/09/rstudio-v1-4-update-whats-new/).
```
Comment (joelostblom) [inline resources_pages/install_ds_stack_windows.md:435]: Could you make changes like this in all three instructions (mac, win, linux?), so that they don't go out of sync with eachother and we forget one of them.

File: resources_pages/install_ds_stack_windows.md [hunk 4]
```diff
@@ -334,14 +334,14 @@ we need to use the `pip` package manager instead of `conda`:
...
-> Note: we will use many more packages than those listed above across the MDS program, however we will manage these using virtual environments (which you will learn about in DSCI 521: Platforms for Data Science).
+> **Note**: we will use many more packages than those listed above across the MDS program, however we will manage these using virtual environments (which you will learn about in DSCI 521: Platforms for Data Science).
 
 ### JupyterLab setup
 
 We will be using the Jupytext Python package and the JupyterLab git extension to facilitate using Jupyter notebooks with Git & GitHub. Install them via the following commands:
 
 ```
-conda install -y nodejs=10.*
+conda install -y nodejs=15.*
```
Comment (joelostblom) [inline resources_pages/install_ds_stack_windows.md]: Could you change this the same way we changed  it for Mac in armands PR? And also add it to the linux doc?

**Review by joelostblom — COMMENTED** [review]
Thanks @flor14 ! See my comments for a few specific things before we merge.

File: resources_pages/install_ds_stack_windows.md (hunk 4 above)
Comment (flor14) [reply resources_pages/install_ds_stack_windows.md]: This phrase has not an example now, as we are not using `conda install -y`, but it is OK to keep it I guess.

> `conda` will show you the packages that will be downloaded,
> and you can press enter to proceed with the installation.
> If you want to answer `yes` by default and skip this confirmation step,
> you can replace `conda install` with `conda install -y`.

**Review by joelostblom — APPROVED** [review]
lgtm! Thanks @flor14 !

File: resources_pages/install_ds_stack_windows.md (hunk 4 above)
Comment (joelostblom) [reply resources_pages/install_ds_stack_windows.md]: I will follow up on this separately if needed
